client/src/sagas/watchers/peerReviewsForUser.js

```diff
@@ -0,0 +1,21 @@
+import { takeLatest, call, put } from 'redux-saga/effects'
+import { LOAD_ALL_USER_PEER_SAGA } from '../../actions/actionTypes.js'
+import { loadAllUserPeerReviews } from '../../api/peerReviewApi'
+import {
+  setAllPeerForUser,
+  setAllPeerForUserError
+} from '../../actions/peerReviewAction'
+
+function* workerLoadUserPeerReviewSaga() {
+  try {
+    const peerReviews = yield call(loadAllUserPeerReviews)
+    yield put(setAllPeerForUser(peerReviews))
+  } catch (e) {
+    console.log(e)
```

Comment:
console log required?

Reply:
@RakshandaWankhede  has fixed this in separate branch. ..